.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature Request
+about: Tell us about something you'd like llvmlite to support. Not for asking general questions - see below.
+
+---
+
+---
+
+<!--
+
+Thanks for opening an issue! To help the Numba team handle your information
```

Comment:
Same Q RE Numba vs llvmlite as above

Reply:
Also, this text probably needs an update to ask users to explicitly check for similar feature requests (seems like it's a copy paste error on Numba's original template?!).